index.html

```diff
@@ -620,6 +659,10 @@
         <h2>
           <code>MediaTrackSettings</code> dictionary
         </h2>
+        <p>
+          <a>MediaTrackSettings</a> dictionary represents the <a>Settings</a>
```

Comment:
<a>MediaTrackSettings</a> partial dictionary extends the original <a>MediaTrackSettings</a> dictionary. It represents the <a>Settings</a> of a <a>MediaStreamTrack</a> object.